spec/labor/badge_rewarder_spec.rb

```diff
@@ -181,4 +181,22 @@
       expect(third_user.reload.badge_achievements.size).to eq(1)
     end
   end
+
+  describe "::award_badges" do
+    let!(:badge) { create(:badge, title: "one-year-club") }
+    let!(:user) { create(:user) }
+    let!(:user2) { create(:user) }
```

Comment:
I think let without the `!` is fine here

Reply:
Why create records 2 times when it's possible to create them once?

Reply:
`let!` is a shortcut for `before { }` which means that the user will be created before each test. `let` without the bang memoizes stuff so it will create only one user. 

Here's a good explanation: https://mixandgo.com/learn/let-vs-instance-variables-in-rspec

Reply:
I supposed that `let!` would be cached between the examples 🤯